libs/dh/shared/feature-flags/README.md

```diff
@@ -1 +1,60 @@
 # DataHub feature flags
+
+You can disable functionality for specified environments: `dev, test, prod.`
+
+<mark>Notice: feature flags treat the pre-prod environment as prod.</mark>
+
+Feature flags are not supposed to live long; therefore, they have a **maximum lifetime of 14 days** (If a feature flag is older than 14 days, an automated test will fail).
+
+## Create a feature flag
+
+Feature flags are located here: `libs/dh/shared/feature-flags/src/lib/feature-flags.ts` -> Find the constant: `dhFeatureFlagsConfig`
+
+The feature flags require two properties, `created` and `disabledEnvironments`.
+
+**Example:**
+
+```ts
+'example-feature-flag': {
+    created: '01-01-2022',
+    disabledEnvironments: [DhAppEnvironment.prod],
+  }
+```
```

Comment:
nitpick:suggestion: let's use the same feature flag name mentioned later in the doc.

```suggestion
```ts
'my-awesome-feature-flag': {
  created: '01-01-2022',
  disabledEnvironments: [DhAppEnvironment.prod],
}
```
```